exporters/ostream/include/opentelemetry/exporters/ostream/span_exporter.h

```diff
@@ -110,11 +110,15 @@ class OStreamSpanExporter final : public sdktrace::SpanExporter
     size_t i    = 1;
     for (auto kv : map)
     {
-      sout_ << kv.first << ": ";
+      if (i == 1)
```

Comment:
Starts `i` from `1` seems not intuitive, perhaps make it starts from `0`?

Reply:
Yes, I did realize this function needs revisit, will fix it.

Reply:
thanks, have removed the use of this index variable.